start_cerberus.py

```diff
@@ -126,24 +141,40 @@ def main(cfg):
                 logging.info("Failed pods and components")
                 for namespace, failures in failed_pods_components.items():
                     logging.info("%s: %s", namespace, failures)
+                if slack_integration:
+                    failed_namespaces = ", ".join(list(failed_pods_components.keys()))
+                    slackcli.post_message_in_slack("In iteration %d, Cerberus found issues in "
```

Comment:
Can we include the info about cluster api? This way, we can differentiate the messages when running the tool on multiple clusters.

Reply:
Yeah, I will add it.